src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write code here
+const tableHeader = document.querySelector('tr');
+const tableBody = document.querySelector('tbody');
+const toNumber = (string) => {
+  if (typeof string === 'string') {
+    const clearNum = string.replace(/[$, ]/g, '');
+
+    return parseInt(clearNum);
```

Comment:
Using 'parseInt' without specifying the radix (the second parameter) can lead to unexpected results if the input string begins with '0'. It's a good practice to always specify the radix, usually 10 for decimal numbers.